src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -103,6 +104,11 @@ export const NavBar: React.FC<NavBarPropsType> = ({
     location: { pathname },
     push: openUrlUsingRouter,
   } = useHistory();
+
+  const { sovereignVenue } = useRelatedVenues({
+    currentVenueId: venueId,
+  });
```

Comment:
You can directly use `useSovereignVenueId` hook here

Reply:
fixed